docs/getting-started/os-support.rst

```diff
@@ -1,11 +1,16 @@
 OS Support by Platform and Version
 ==================================
 
-The following matrix shows which Operating Systems, Platforms, and Containers / Instance Engines are supported with which versions of Scylla.
+The following matrix shows which Linux Distribution, Containers, and images are supported with which versions of ScyllaDB.
```

Comment:
Linux distributions (plural, and only the proper noun Linux is capitalized, "distributions" is not).
containers (uncapitalized).